README.md

```diff
@@ -1,21 +1,36 @@
 # Welcome to DuckDuckHack!
 
-![](http://docs.duckduckhack.com/assets/hack search engine.png)
+We're a community dedicated to improving search results with open source, "Instant Answers", that solve searches in few or zero clicks.
 
-We're a community dedicated to improving search results with open source, "Instant Answers." Instant Answers use the Web's best APIs and data to solve searches in few or zero clicks. Together and openly, we can create the best search engine for every type of search.
 
-**The DuckDuckHack community and DuckDuckGo staff are working together on a mission to create the best search engine for programmers. To fully focus there, we are only accepting Pull Requests and Issues related to the [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).**
+### The Programming Mission
+The current focus of DuckDuckHack is to create the best search engine for programmers. For now, we are **only accepting Pull Requests and Issues related to the Programming Mission**.
 
-[**Join the Programming Mission!**](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53)  
-*Your help and expertise can have a huge impact on improving the search experience for developers and DuckDuckGo users around the world!*
 
-More Resources:   
-*[Full documentation](http://docs.duckduckhack.com)*  
-*[Instant Answers in Production](https://duck.co/ia)*  
-*[Instant Answers in Beta](https://beta.duckduckgo.com/?q=test&t=hc&ia=iatesting&iax=1)*  
+## Get Started
+Your help and expertise can improve the search experience for millions of DuckDuckGo users around the world!
 
-Instant Answers appear at the top of search results...
-![](http://docs.duckduckhack.com/assets/python_apply_at_DuckDuckGo.png)
+There are two ways you can get involved:
 
-Attribution is shown for developers who create or improve them...
-![](http://docs.duckduckhack.com/assets/attribution_example_1.png)
+1. [**Create new Fatheads, and improve existing ones**](https://github.com/duckduckgo/zeroclickinfo-fathead/issues?q=is%3Aopen+is%3Aissue+label%3A"Mission%3A+Programming")
+    - **Note**: This will require knowledge of Perl, Python, Ruby, or JavaScript (Node.js)
```

Comment:
Perhaps we should link to the Docs page (https://docs.duckduckhack.com/resources/fathead-overview.html#fathead-instant-answers) which mentions which languages are supported.

The 'require' here feels a bit strong, implying prior knowledge is required (but they can learn through development).